rasa/core/training/interactive.py

```diff
@@ -1130,9 +1136,24 @@ async def _validate_action(
 
     Returns `True` if the prediction is correct, `False` otherwise."""
 
-    question = questionary.confirm(f"The bot wants to run '{action_name}', correct?")
+    if action_name == ACTION_UNLIKELY_INTENT_NAME:
+        question = questionary.confirm(
+            f"The bot wants to run '{action_name}' "
+            f"to indicate that the last user message was unexpected "
+            f"at this point in the conversation. "
+            f"Check out UnexpecTEDIntentPolicy "
+            f"({DOCS_URL_POLICIES}#unexpected-intent-policy) "
+            f"to learn more."
+        )
+    else:
+        question = questionary.confirm(
+            f"The bot wants to run '{action_name}', correct?"
+        )
 
-    is_correct = await _ask_questions(question, conversation_id, endpoint)
+    is_correct = (
```

Comment:
I just tried this, and it still registers the `action_unlikely_intent` even when I select `n`.

Reply:
Also, the `question` is not really a question.

Reply:
@alwx

Reply:
This is what we actually decided to do for the sake of consistency. If we decide that it's a bad idea then we can also switch back to just using a simple "press any button to continue" message.

Reply:
@alwx I think @JEM-Mosig is referring to the fact that `action_unlikely_intent` still shows up as a registered action in the conversation so far even when you press `n`.

<img width="1428" alt="Screenshot 2021-07-05 at 14 09 07" src="https://user-images.githubusercontent.com/8708249/124469399-f3903500-ddb7-11eb-9a08-6560bac0b639.png">


Regarding the message:

> The bot wants to run 'action_unlikely_intent' to indicate that the last user message was unexpected at this point in the conversation. Check out UnexpecTED IntentPolicy to learn more. (Y/n)


I think the information giving more context into `action_unlikely_intent` is def useful. However, `Y/n` is the confusing part. User doesn't know what are they answering `Y/n` to. If you append something like `Is the action correct (Y/n)` at the end then that would be better.

Reply:
Yes, this is what I meant.

Reply:
@alwx Please let me know whenever you have addressed this, so that I can test it out locally. Thanks

Reply:
Yes, now it just shows a prompt and there is no "Yes/No" choice there.

Reply:
This is what I see:

<img width="1438" alt="Screenshot 2021-07-06 at 09 42 52" src="https://user-images.githubusercontent.com/8708249/124561841-f6446600-de5b-11eb-8260-59256a9867cd.png">

Can we please add a description for the prompt in the next line, something like - `Is that correct? (Y/n)`. Right now, I as a user don't know what to press / how to proceed?

Reply:
Not sure if you have added a fix for this already but `action_unlikely_intent` still shows up as a registered action even when I press `n`.

<img width="1431" alt="Screenshot 2021-07-06 at 09 46 15" src="https://user-images.githubusercontent.com/8708249/124562332-87b3d800-de5c-11eb-9ed5-40905ef4e448.png">

Reply:
What exactly needs to be happening in this case? Do we need to use the correction mechanism for `action_unlikely_intent`? What needs to be happening when use presses `n`?

Reply:
If they press `y`: `action_unlikely_intent` gets registered and ensemble is triggered for another action as usual.
If they press `n`: `action_unlikely_intent` should not get registered but no correction mechanism needs to be triggered. The ensemble should be triggered for another action instead.

Whether `action_unlikely_intent` was registered or not does not change the fact that it should not dumped to a story at all.
Does that help clarify?

Reply:
How can the conversation progress if `action_unlikely_intent` is not registered? Won't it be predicted again in this case?

Reply:
By not registered I meant not showing up in the `Chat History` but still added to the tracker so that other policies can predict. Would that make the implementation too complex?
If yes, another alternative is that we show the message ```The bot **triggered** `action_unlikely_intent` to indicate .....``` and then we don't prompt the user to continue with a y/n. If `y`, query the ensemble for another action, if `n` show the menu.

Reply:
Oh I actually have an idea, one sec

Reply:
@dakshvar22 is that correct? In the first case `action_unlikely_intent` does get added, in the second — it doesn't. In both cases dumped stories file contains no `action_unlikely_intent`.

https://user-images.githubusercontent.com/911127/124759764-4e4c9c80-df30-11eb-8731-3739e258d956.mov

Reply:
Yes, looks perfect. 😄